tests/migration_test.go

```diff
@@ -4552,8 +4553,72 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 			}
 		})
 	})
+	Context("ResourceQuota rejection", func() {
+		It("Should contain condition when migrating with quota that doesn't have resources for both source and target", func() {
+			vmiRequest := resource.MustParse("200Mi")
+			vmi := libvmi.NewCirros(
+				libvmi.WithNetwork(v1.DefaultPodNetwork()),
+				libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+				libvmi.WithResourceMemory(vmiRequest.String()),
+			)
+
+			vmiRequest.Add(resource.MustParse("50Mi")) //add 50Mi memoryOverHead to make sure vmi creation won't be blocked
+			enoughMemoryToStartVmiButNotEnoughForMigration := services.GetMemoryOverhead(vmi, runtime.GOARCH, nil)
+			enoughMemoryToStartVmiButNotEnoughForMigration.Add(vmiRequest)
+			resourcesToLimit := k8sv1.ResourceList{
+				k8sv1.ResourceMemory: resource.MustParse(enoughMemoryToStartVmiButNotEnoughForMigration.String()),
+			}
+
+			By("Creating ResourceQuota with enough memory for the vmi but not enough for migration")
+			resourceQuota := newResourceQuota(resourcesToLimit, testsuite.GetTestNamespace(vmi))
+			_ = createResourceQuota(resourceQuota, 90*time.Second)
```

Comment:
This can be simplified as follows:
1. Create VMI
2. Create a quota allowing 0 or minimum
3. Run migration and expect failure

Reply:
Fair, you might want to fetch the pod and use the memory request then to hide the magic calculation.